perf/stability/alertmanager/prometheusrule.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: prometheus
+    role: prometheus-example-rules
+  name: prometheus-example-rules
+  namespace: istio-prometheus
+spec:
+  groups:
+    - name: ./workload.rules
```

Comment:
nit:  some rules start with `./infra.rules`, and some do not have `./`

Reply:
all alerting rules start with `./`